pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -576,6 +579,20 @@ The last argument is a JSONB object containing the following optional fields:
 			volatility.Volatile,
 		),
 	),
+	"crdb_internal.show_tenant_capabilities_cache": makeBuiltin(
+		tree.FunctionProperties{},
+		makeGeneratorOverload(
+			tree.ParamTypes{},
+			showTenantCapabilitiesCacheType,
+			func(ctx context.Context, evalCtx *eval.Context, args tree.Datums) (eval.ValueGenerator, error) {
+				return &showTenantCapabilitiesCacheGenerator{
+					p: evalCtx.Planner,
+				}, nil
+			},
+			`Returns the in-memory tenant capability cache for this node`,
+			volatility.Stable,
```

Comment:
I am not sure it matters a ton here, but is this volatility correct? The documentation for stable includes "is guaranteed to return the same results given the same arguments whenever it is evaluated within the same statement."

But I'm not sure that is _really_ the case here since our read of the cache isn't bound to the transaction (and can't be).

Reply:
Switched to a vtable so this no longer applies.